parsec/core/fs/workspacefs/sync_transactions.py

```diff
@@ -240,6 +240,23 @@ async def synchronization_step(
         # Fetch and lock
         async with self.local_storage.lock_manifest(entry_id) as local_manifest:
 
+            # Go through the parent chain
+            current_manifest = local_manifest
+            while not is_workspace_manifest(current_manifest):
+                parent_manifest = await self.local_storage.get_manifest(current_manifest.parent)
+                parent_manifest = cast(LocalFolderishManifests, parent_manifest)
+
+                # The entry is not confined
+                if current_manifest.id not in parent_manifest.confined_entries:
+                    current_manifest = parent_manifest
+                    continue
+
+                # Send synced event
+                self._send_event(
+                    CoreEvent.FS_ENTRY_CONFINED, entry_id=entry_id, cause_id=parent_manifest.id
```

Comment:
so this event is triggered every time an entry should have been synchronized but is filtered ? I guess it's useful because we can listen to `FS_ENTRY_SYNCED `and `FS_ENTRY_CONFINED `and get the same behavior as when we used to listen to `FS_ENTRY_SYNCED` alone